instrument-starters/opentracing-spring-cloud-mongo-starter/src/main/java/io/opentracing/contrib/spring/cloud/mongo/MongoTracingAutoConfiguration.java

```diff
@@ -18,7 +18,6 @@
 import io.opentracing.Tracer;
 import io.opentracing.contrib.mongo.TracingMongoClient;
 import io.opentracing.contrib.spring.tracer.configuration.TracerAutoConfiguration;
-import java.net.UnknownHostException;
```

Comment:
no change here

Reply:
Had to remove the "throws exception" here due to
![image](https://user-images.githubusercontent.com/5640930/44230685-c14b4480-a193-11e8-8eab-9eded09ceef2.png)

Reply:
then it's fine :+1: